### PR TITLE
F/va 128

### PR DESCRIPTION
**Pull request by stanbellcom**

### CODE REVIEW

File: public_html/profiles/valhalla/modules/valghalla/modules/valghalla_external_server/valghalla_external_server.module [hunk 1]
```diff
@@ -5,11 +5,14 @@
...
  */
 
+use ValghallaVolunteers\ValghallaVolunteersLogger;
+
 /**
  * Implements hook_node_insert().
  */
 function valghalla_external_server_node_insert($node) {
   if (in_array($node->type, array('volunteers'))) {
+    ValghallaVolunteersLogger::info(sprintf('Creating new volunteer %s (nid: %d, uuid: %s)', $node->title, $node->nid, $node->uuid));
```
Comment (andriyun) [inline public_html/profiles/valhalla/modules/valghalla/modules/valghalla_external_server/valghalla_external_server.module]: Drupal has line own function to work with place holders https://api.drupal.org/api/drupal/includes%21bootstrap.inc/function/format_string/7.x
Please it here and all following statements lie this.

File: public_html/profiles/valhalla/modules/valghalla/modules/valghalla_volunteers/valghalla_volunteers.module [hunk 2]
```diff
@@ -5,9 +5,12 @@
  * Code for the valghalla volunteers feature.
  */
 
+use ValghallaVolunteers\ValghallaVolunteersLogger;
+
 include_once 'valghalla_volunteers.features.inc';
 require_once 'valghalla_volunteers.inc';
 module_load_include('inc', 'valghalla_volunteers', 'includes/valghalla_volunteers.utils');
+module_load_include('php', 'valghalla_volunteers', 'includes/ValghallaVolunteersLogger.class');
```
Comment (andriyun) [inline public_html/profiles/valhalla/modules/valghalla/modules/valghalla_volunteers/valghalla_volunteers.module]: Please use composer friendly way to include files like this.
autoload file + `use` direcritive 
See examples in monolog contrib module

File: public_html/profiles/valhalla/modules/valghalla/modules/valghalla_external_server/valghalla_external_server.module (hunk 1 above)
Comment (stanbellcom) [reply public_html/profiles/valhalla/modules/valghalla/modules/valghalla_external_server/valghalla_external_server.module]: format_string is that a critical change to you?
I don't see any reason for switching to to. And the fact that drupal has some of it's own format function, is not an argument to use it.

However i see counter arguments to not use it:
- The line code becomes longer, or we need to add more variables before it.
- We will also need to change other log messages (synch related) that are using sprintf function.

Comment (andriyun) [reply public_html/profiles/valhalla/modules/valghalla/modules/valghalla_external_server/valghalla_external_server.module]: Nope it's not critical.

File: public_html/profiles/valhalla/modules/valghalla/modules/valghalla_volunteers/valghalla_volunteers.module (hunk 2 above)
Comment (stanbellcom) [reply public_html/profiles/valhalla/modules/valghalla/modules/valghalla_volunteers/valghalla_volunteers.module]: after discussion in skype, ti was decided to drop it

**Pull request merged**